app/src/components/widgets/singleLevelWidgets/tables/projectActivity/activities/notifications.jsx

```diff
@@ -35,18 +30,10 @@ const messages = defineMessages({
     id: 'Notifications.notifications',
     defaultMessage: 'E-mail notifications',
   },
-  [UPDATE_NOTIFICATIONS]: {
+  updated: {
     id: 'Notifications.updateEmail',
     defaultMessage: 'updated',
   },
-  [SWITCH_ON_NOTIFICATIONS]: {
-    id: 'Notifications.onEmail',
-    defaultMessage: 'configured',
-  },
-  [SWITCH_OFF_NOTIFICATIONS]: {
-    id: 'Notifications.offEmail',
-    defaultMessage: 'switched off',
-  },
 });
 
 @injectIntl
```

Comment:
What do you think about rewriting this component as a pure function?

Reply:
I like the idea, but it's not relevant for the scope of this PR and also it's not necessary